### DISCUSSION TITLE
Nginx 307 error in GCP

### DISCUSSION THREAD

**Discussion by krshnn** [Security-Onion-Solutions/securityonion#6451] [Unsupported Versions] [answered]
I was trying to install security onion on GCP recently, installation went well and all containers were running fine. when I tried to access the interface through public IP, it returned  a 307 Temporary Redirect error. Same error while running curl command locally. Has anyone else faced this issue before? or was I doing something wrong? I have also installed portainer in the instance and was able to access the interface through public ip. It doesnt have reverse proxy though

**Comment by dougburks** [comment]
When the installer asked the following question, how did you respond?
![image](https://user-images.githubusercontent.com/1659467/145038972-ea360ffe-b982-4bcd-a7e8-9c2174bd3a79.png)

**Comment by krshnn** [comment]
I gave IP.

**Comment by krshnn** [comment]
I was able to access the console through another VM internally, This happened when I tried through the public IP (wasn't a good idea). Later I closed the FW as it was attracting unwanted traffic.

**Comment by dougburks** [comment, marked as answer]
When the installer asks the question above and you choose the IP option, then the instance will only be accessible via the actual IP address assigned to the management interface.

We don't have documentation for GCP specifically, but our Amazon and Azure pages both address this:
> This section does not cover network connectivity to the Security Onion node. This can be achieved through configuring an external IP for the node’s management interface, or through the use of a VPN connection via OpenVPN. For more details about VPN connections, please see https://medium.com/@svfusion/setup-site-to-site-vpn-to-aws-with-pfsense-1cac16623bd6.

https://docs.securityonion.net/en/2.3/cloud-ami.html
https://docs.securityonion.net/en/2.3/cloud-azure.html

**Reply by krshnn** [reply]
Thank you! I'll try this out.
